### PR TITLE
Fix/nation list label status

### PR DESCRIPTION
**Pull request by albertoestarrona**
> Please check you are not duplicating an existing pull request...

#### Changes
Solves the status color in label (NationsList)

### CODE REVIEW

File: src/utils/nations.js [hunk 1]
```diff
@@ -45,4 +49,20 @@ export function nationIsValid(nation) {
   if (_.isEmpty(nation.governanceService)) return false;
 
   return true;
-}
\ No newline at end of file
+}
+
+// @todo Use Pangea libs implementation.
+export function resolveListStyle(nation): nationStatus {
+  if (nation.created) {
```
Comment (seland) [inline src/utils/nations.js]: Please do not copy the implementation of resolveStatus, but reuse it instead. So, if it's changed eventually, we don't need to update both methods.

File: src/utils/nations.js [hunk 2]
```diff
@@ -45,4 +49,20 @@ export function nationIsValid(nation) {
...
+    return Colors.Green;
+  }
+  if (nation.txHash) {
+    return Colors.Amber;
+  }
+
+  return Colors.listItemTextState;
+}
+
+const styles = MediaQueryStyleSheet.create({
```
Comment (seland) [inline src/utils/nations.js]: Is it used anywhere? Consider removing.

File: src/utils/nations.js [hunk 3]
```diff
@@ -1,4 +1,8 @@
 import _ from 'lodash';
+import GlobalStyles from '../global/Styles';
```
Comment (seland) [inline src/utils/nations.js]: It seems to be unused.

File: src/utils/nations.js [hunk 4]
```diff
@@ -1,4 +1,8 @@
...
+import {MediaQueryStyleSheet} from "react-native-responsive";
```
Comment (seland) [inline src/utils/nations.js]: It seems to be unused.

File: src/screens/NationsScreen/NationsListScreen/index.js [hunk 5]
```diff
@@ -58,6 +58,7 @@ class NationsListScreen extends Component {
             return (<NationListItem text={nation.nationName}
                                     onPress={this.props.onSelectItem}
                                     status={i18n.t(`enums.nation.status.${resolveStatus(nation)}`)}
+                                    statusColor={resolveListStyle(nation)}
```
Comment (seland) [inline src/screens/NationsScreen/NationsListScreen/index.js]: If you return a color name, please change the method name, so it doesn't look as if it returns style object

File: src/utils/nations.js (hunk 2 above)
Comment (albertoestarrona) [reply src/utils/nations.js]: Removed

File: src/utils/nations.js (hunk 3 above)
Comment (albertoestarrona) [reply src/utils/nations.js]: Removed

File: src/utils/nations.js (hunk 4 above)
Comment (albertoestarrona) [reply src/utils/nations.js]: Removed

File: src/screens/NationsScreen/NationsListScreen/index.js (hunk 5 above)
Comment (albertoestarrona) [reply src/screens/NationsScreen/NationsListScreen/index.js]: Changed the name to `statusColor`

File: src/utils/nations.js (hunk 1 above)
Comment (albertoestarrona) [reply src/utils/nations.js]: Done some refactor

**Comment by albertoestarrona** [comment]
Closed because #241 covers all this updated

**Pull request closed without merging**